### ISSUE TITLE
Nuke.MSBuildTasks.CodeGenerationTask fails when upgrading to 0.24.9

### ISSUE THREAD

**Issue by jnyrup** [nuke-build/nuke#480]
### Description
I tried upgrading Fluent Assertions from 0.24.7 to 0.24.9, but Visual Studio gives me the error below.
I tried upgrading to 0.24.8 first, and that didn't gave any errors.

### Steps to reproduce
1. git clone https://github.com/fluentassertions/fluentassertions.git 
2. Checkout develop
3. Change `Build/_build.csproj` to use 0.24.9 instead of 0.24.7
4. Open `FluentAssertions.sln` in Visual Studio 2019
5. I get the error above

### Relevant log output

```
C:\Users\JonasNyrup\.nuget\packages\nuke.common\0.24.9\build\netcoreapp2.1\Nuke.MSBuildTasks.targets(43,5): 
error MSB4062: 
The "Nuke.MSBuildTasks.CodeGenerationTask" task could not be loaded from the assembly C:\Users\JonasNyrup\.nuget\packages\nuke.common\0.24.9\build\netcoreapp2.1\\Nuke.MSBuildTasks.dll. 
Could not load file or assembly 'System.Runtime, Version=4.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. 
The system cannot find the file specified. 
Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.
```

**Comment by matkoch** [comment]
@jnyrup I just tried and don't see any issues when just running `nuke` as a command. Is it only in VisualStudio ?

**Comment by jnyrup** [comment]
> running `nuke` as command
Is that running `build.ps1`?
That runs fine.

Tried deleting the .vs folder and running `devenv /SafeMode` to see if it might be some cached stuff, but it didn't change the outcome.

So right now, I've been only been able to trigger it from Visual Studio Professional 16.5.4.
I also have a Visual Studio 16.6 preview installation where it also fails.

**Comment by jnyrup** [comment]
I can reproduce it outside Visual Studio, using 

```
msbuild _build.csproj
```

**Comment by sm-g** [comment]
Same for me, cannot write code in VS now, because all of yellow triangle on Packages all `using ` cannot be resolved and removes on code cleanup.

**Comment by matkoch** [comment]
@sm-g not sure what _same_ is supposed to mean here. That sounds like a complete different scenario.

**Comment by sm-g** [comment]
Same error from msbuild when build csproj. Triangles just an indicator.

**Comment by gliljas** [comment]
I get the same error. Both yellow triangle and unable to build in VS. 

Maybe worth noting are the double slashes in "build\netcoreapp2.1\\Nuke.MSBuildTasks.dll"

Downgrading to 0.24.8 works for now.

**Comment by Bomret** [comment]
I have the same problem. I am using VS Code with the C# Extension (Omnisharp). Here is the log:

[omnisharp.log](https://github.com/nuke-build/nuke/files/4515663/omnisharp.log)

**Comment by andrecarlucci** [comment]
Same problem here. Going back to version 0.24.8 fixes it for now.

`Error	MSB4062	The "Nuke.MSBuildTasks.CodeGenerationTask" task could not be loaded from the assembly C:\Users\André Carlucci\.nuget\packages\nuke.common\0.24.9\build\netcoreapp2.1\\Nuke.MSBuildTasks.dll. Could not load file or assembly 'System.Runtime, Version=4.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.	_build	C:\Users\André Carlucci\.nuget\packages\nuke.common\0.24.9\build\netcoreapp2.1\Nuke.MSBuildTasks.targets	43	`

**Comment by matkoch** [comment]
Should be fixed in latest release. And to everyone who has been adding their "individual" log file, which basically showed the same error: next time, please think about whether your message gives any value. GitHub has a +1 feature, I'm sure you know about that. I have notifications enabled for this project, and you're not doing anything than stressing me and potentially forcing me to disable them. you can decide if that's in your interest.

**Comment by Bomret** [comment]
I can understand that you prob got a number of notifications. But if one's build system stops working, people tend to get nervous and try to help out by providing their information so the maintainer has more than one source of info.
If you try to get a certain flow of issue contribution, I advise to put up a CONTRIBUTING.md which states that as every project handles these differently.

You came out quite rude to be honest. Dont' know "if that's in your interest".

**Comment by sm-g** [comment]
There is already https://github.com/nuke-build/.github/blob/master/.github/CODE_OF_CONDUCT.md#our-standards

But declaring standarts and following is not the same. I see only silent closing of my issues. Lesson learned, of course , will never ask anything here.

**Comment by matkoch** [comment]
@sm-g you’ve bern rude more than once. Which was not only pointed out by me. The standards you referenced are supposed to work both ways.

**Comment by matkoch** [comment]
@Bomret ich finds etwas vermessen daraus jetzt so einen aufstand zu machen. ich hab die comments geloescht, weil es duplication war, und ich habe erklaert warum. jetzt wieder “den user” und meine angebliche unfreundlichkeit in den vordergrund zu stellen, zeigt mal wieder wie kaputt open source ist. ich teile hier ueber 3 jahre meiner arbeit mit allen, aber wehe man macht mal etwas, was nicht den eigenen vorstellungen passt.

**Comment by matkoch** [comment]
@Bomret und dann wird einfach mal geherzt bei situation die man gar nicht kennt. echt grandios. sehr gute erste contribution.

**Comment by Bomret** [comment]
@matkoch es ging nicht um das Löschen der Kommentare. Ich kann sehr gut verstehen, dass Du sie gelöscht hast, da sie Dir keinen sinnvollen Mehrwert gebracht haben und Du Signal/Noise Ratio verbessern wolltest.
Es geht mir mit meinem Kommentar darum, dass Du denen (auch mir), bei denen das Buildsystem plötzlich nicht mehr funktionierte (was in der Regel ein ziemlich kritischer Bereich ist) und die helfen wollten, dem Problem schnellstmöglich auf die Spur zu kommen, mit einer sehr unangemessenen Sprache quasi das sinnlose Spammen des Tickets vorgeworfen hast. Dabei dann auch noch anzumerken, dass Du in aufgrund solcher Fälle dann zukünftig Deine Notifications ausschaltest und sich dann quasi alle, die kritische Bugs melden und auf schnelle Hilfe hoffen, bei denen dafür bedanken können, ist dann noch die süffisante Kirsche auf der ganzen Reaktion. Woher sollten wir wissen, dass die Info Dir nicht zusätzlich hilft?

Ich weiß, dass es nicht immer leicht ist Maintainer zu sein und niemand muss mit unagemessenen Forderungen oder Bullshit-Kommentaren leben. Aber hier in diesem Fall ist so eine ad-hominem Blame Game Sprachwahl absolut unangemessen. Vor allem wenn man seinem eigenen Projekt einen Code of Conduct gibt, welcher solches Verhalten unterbinden soll, worauf @sm-g auch zurecht hingewiesen hat (daher mein Herz für seinen Kommentar). Für einen bestimmten Issue Flow auch noch mal mein Hinweis auf eine `CONTRIBUTING.md`.

Eine angemessene Reaktion wäre: Unnötige Kommentare löschen und dann zum Beispiel sowas posten wie:

> Thanks everyone trying to help here. I deleted some comments that contained no new info so that I can focus better on the problem and keep the signal/noise ratio of this thread. The bug was fixed in version 0.24.10 and I hope no one runs into this problem again! In the future you can help me prioritise things by adding a +1 to the original comment. If I need more info, I'll ask for it then.
